docs/2.8.0/docs/canton/usermanual/persistence.rst

```diff
@@ -61,6 +61,50 @@ This snippet shows how ssl can be configured for Postgres. You can find more inf
 
 Note that all configuration properties for the database will be propagated to the Ledger API JDBC URL.
 
+.. _postgres-performance-tuning:
+
+Performance 
+~~~~~~~~~~~
+
+Please note that your Postgres database setup will require appropriate tuning to achieve the desired performance. Canton 
+is a rather database heavy process. This section should give you a starting point for your tuning efforts. You may want to consult 
+the :ref:`troubleshooting section <how_to_measure_db_performance>` on how to analyse whether the database is a limiting factor. 
+
+This guide will give you a starting point. Ultimately, every use case is different and the exact resource requirements can not 
+be predicted, but have to be measured.
+
+First, ensure that the database you are using is appropriately sized for your use case. The number of cores depends on your 
+throughput requirements. The rule of thumb is 
+    * 1 db core per 1 participant core.
+    * 1 participant core for 30-100 ledger events (depends on the complexity of the commands).
+
+The memory requirements depend on your data retention period and the size of the data you are storing. Ideally, you monitor the 
+database index cache hit / miss ratio. If your instance needs to keep on loading indexes from the disk, the performance will be 
+poor. It might make sense to start with 128GB, run a long-running scale & performance test and `monitor the cache hit / miss ratio <https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STATIO-ALL-INDEXES-VIEW>`__`
```

Comment:
```suggestion
database index cache hit/miss ratio. If your instance needs to keep on loading indexes from the disk, performance suffers. It might make sense to start with 128GB, run a long-running scale & performance test, and `monitor the cache hit/miss ratio <https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STATIO-ALL-INDEXES-VIEW>`__`
```

Reply:
fixed